src/server/_query.py

```diff
@@ -474,8 +474,11 @@ def apply_lag_filter(self, history_table: str, lag: Optional[int]) -> "QueryBuil
 
     def apply_issues_filter(self, history_table: str, issues: Optional[TimeValues]) -> "QueryBuilder":
         if issues:
-            self.retable(history_table)
-            self.where_integers("issue", issues)
+            if issues == ["*"]:
+                self.retable(history_table)
+            else:
+                self.retable(history_table)
+                self.where_integers("issue", issues)
```

Comment:
```suggestion
            self.retable(history_table)
            if issues != ["*"]:
                self.where_integers("issue", issues)
```
you dont have to repeat the `retable()` line, and the `where_integers()` call is better done under the `if` instead of the `else` (by reversing the comparison).